### DISCUSSION TITLE
Where can i change the default sampler on load?

### DISCUSSION THREAD

**Discussion by runew0lf** [vladmandic/sdnext#113] [Q&A]
Its currently set to UNIPC and i'd like to switch it to euler A on load?

**Comment by vladmandic** [comment]
edit that setting in `ui-config.json`

**Reply by runew0lf** [reply]
awesome buddy thankyou! <3

**Reply by drax-xard** [reply]
For anyone else wondering the line in ui-config.json is:
`"txt2img/Sampling method/value":`

**Reply by vladmandic** [reply]
use settings -> set defaults, thats better way than editing ui-config.json since it was added.

**Reply by brknsoul** [reply]
It's actually called "User interface defaults" near the bottom of the list of Settings pages.

Simply set your UI how you want it (including prompts and negs, if you like), then head to the above Settings page and click "View changes", confirm that the listed changes are accurate, then click "Set new defaults".
